python/tests/test_format.py

```diff
@@ -25,7 +25,7 @@ def test_bad_mid(mid: int) -> None:
         Message(MessageType.REQUEST, b"hello", mid, [])
 
 
-@pytest.mark.parametrize("name", [b"", b"0", b"underscores_not_allowed", b"t\xFF"])
+@pytest.mark.parametrize("name", [b"", b"0", b"underscores_not_allowed", b"t\xff"])
```

Comment:
Interesting, I wonder why it prefers/requires lowercase. Is that part of a style guide?

Reply:
It's not part of PEP 8. But I guess it enforces *something* just for consistency.